### DISCUSSION TITLE
PNP Result Webpart shows the following error message, when SharePoint Search is choosen: [HTTP]:400 - [CorrelationId]:2dd193a0-f065-6000-0d9b-fceea3692237 [Version]:16.0.0.23318

### DISCUSSION THREAD

**Discussion by sayuri8** [microsoft-search/pnp-modern-search#2750] [Q&A]
Hi everyone,

Yesterday worked everything fine but if I am checking now it shows the following error message especially when SharePoint Search is choosen and there aren't any settings which I made. I took the standard one. 
[HTTP]:400 - [CorrelationId]:2dd193a0-f065-6000-0d9b-fceea3692237 [Version]:16.0.0.23318

**Comment by kasperbolarsen** [comment]
Please check the API using either SharePoint Query tool or similar and revert.

**Comment by sayuri8** [comment]
Thanks for your fast response! Is this the API: https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/get-to-know-the-sharepoint-rest-service?tabs=csom 

SharePoint Query tool I never heard about it? Does only the admin has the rights or who can revert it?

**Comment by kasperbolarsen** [comment]
Do I have good news for you 😄 Working with search is a pain without the right tooling,  and SharePoint Search Query tool makes it a lot easier 
https://github.com/pnp/PnP-Tools

**Comment by sayuri8** [comment]
Thanks, I will have a look and if I have further questions. I will come back! Do you know the reason why the error message happens? Do I need admin right to revert this?

**Comment by sayuri8** [comment]
Now I had a look into it but not sure what I have to do. I have to download the zip file and run the exe file and then I can work with this tool. Are there any instructions how to use this? Do I have to code something? Sorry, I am new.

**Comment by kasperbolarsen** [comment]
Google is you friend. A quick search will show both a Laura Rogers video and a guide by Antti https://www.koskila.net/using-sharepoint-search-query-tool-investigate-search-index-issues-sharepoint-online/

**Comment by sayuri8** [comment]
Great thanks!
